### PR TITLE
[RI1] Removing outdated comments

### LINKED ISSUE #2110: [RI1] Removing outdated comments
The chapters of the documentation contains large outdated comments. These could be removed for easier navigation in the source files and to avoid maintenance issues.

### PR DESCRIPTION
**Pull request by EditKoselak**
Closes #2110 and changes references to the Reference Model from RM1 to RM.

### CODE REVIEW

File: doc/ref_impl/cntt-ri/chapters/chapter03.md [hunk 1]
```diff
@@ -23,45 +23,17 @@ Any virtual network functions and/or cloud-native network functions must choose
 
 `B/N/C <I opt> . <Flavour> . <S ext> . <A ext>`
 
-B/N<!---/C---> is used to specify the instance type (Basic, Network Intensive<!---, and Compute Intensive--->), different instance types are associated with different acceleartion extensions, network characteristics ([RM1: 4.2.4 Instance Types](../../../ref_model/chapters/chapter04.md#4.2.4)) and instance capabilities([RM1: 4.2.5 Instance Capabilities Mapping](../../../ref_model/chapters/chapter04.md#4.2.5)).
+B/N is used to specify the instance type (Basic, Network Intensive), different instance types are associated with different acceleartion extensions, network characteristics ([RM1: 4.2.4 Instance Types](../../../ref_model/chapters/chapter04.md#4.2.4)) and instance capabilities([RM1: 4.2.5 Instance Capabilities Mapping](../../../ref_model/chapters/chapter04.md#4.2.5)).
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/chapters/chapter03.md:26]: while deleting comments, can we also correct "RM1" to "RM"?

Comment (EditKoselak) [reply doc/ref_impl/cntt-ri/chapters/chapter03.md]: Thank you for the comment! I made the changes and updated the description of the PR with this information.

**Comment by EditKoselak** [comment]
@pgoyal01, who else could review these changes?

**Review by gkunz — APPROVED** [review]
LGTM - only removing comments which have been in here for a while.

**Pull request merged**